src/v1/imports/carts/ecwid.ts

```diff
@@ -0,0 +1,47 @@
+import { QueryStringContext } from "../../../shared/types";
+import { tryParseJSONObject } from "../../../shared/utils/try-parse-json";
+
+const ecwidTracker = ({ appId, retailId }: Pick<QueryStringContext, "appId" | "retailId">) => {
+  if (!window.transactionOrder && !window.transactionItems && !window.email) {
```

Comment:
I think we should separate the guard clause for the email. I am not sure if we get emails all the time. I'm thinking that this might be too strict and some transactions might not get tracked

I'm thinking that maybe we should do something like:

```
IF email address exists 
THEN set user id to email address
```

which makes recording emails an option instead of a necessity